etc/inkscape.profile

```diff
@@ -41,5 +43,6 @@ shell none
 private-dev
 private-tmp
 
+memory-deny-write-execute
```

Comment:
do extensions continue to work?

Reply:
I must admit I only tested a few extensions. And after some more testing (at least on Arch) some need python. To be safe I'll remove memory-deny-write-execute and allow python.

Reply:
Should be fixed in https://github.com/netblue30/firejail/pull/2473.